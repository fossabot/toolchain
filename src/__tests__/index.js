import { logger, metrics, metricsHandler, liveHandler } from '../index';

describe('index', function() {
  it('logger', function() {
    expect(typeof logger).toBe('object');
  });

  it('metrics', function() {
    expect(typeof metrics).toBe('object');
  });

  it('metricsHandler', function() {
    expect(typeof metricsHandler).toBe('function');
  });

  it('liveHandler', function() {
    expect(typeof liveHandler).toBe('function');
  });
});
