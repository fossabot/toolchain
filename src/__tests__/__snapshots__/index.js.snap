// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example would fail if constants corrupt 1`] = `"Something interesting has happened ;-)"`;
